common/v2/types/network.ts

```diff
@@ -45,6 +45,7 @@ export type NetworkId =
 export interface Network {
   id: NetworkId;
   name: string;
+  unit: string;
```

Comment:
use `baseAsset` `ticker` over `unit`

Reply:
This has also been removed, not sure why it wasn't marked as resolved.

Reply:
I'm sorry the code was present in the changes, so I unresolved.
I may have been reading the wrong commit and apologise for the noise.